src/status_im2/contexts/quo_preview/switcher/group_messaging_card.cljs

```diff
@@ -0,0 +1,144 @@
+(ns status-im2.contexts.quo-preview.switcher.group-messaging-card
+  (:require [quo.react-native :as rn]
+            [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]
+            [status-im2.common.resources :as resources]))
+
+(def descriptor
+  [{:label "Title"
+    :key   :title
+    :type  :text}
+   {:label   "Status"
+    :key     :status
+    :type    :select
+    :options [{:key   :read
+               :value :read}
+              {:key   :unread
+               :value :unread}
+              {:key   :mention
+               :value :mention}]}
+   {:label "Counter Label"
+    :key   :counter-label
+    :type  :text}
+   {:label   "Type"
+    :key     :type
+    :type    :select
+    :options [{:key   :message
+               :value :text}
+              {:key   :photo
+               :value :photo}
+              {:key   :sticker
+               :value :sticker}
+              {:key   :gif
+               :value :gif}
+              {:key   :audio
+               :value :audio}
+              {:key   :community
+               :value :community}
+              {:key   :link
+               :value :link}
+              {:key   :code
+               :value :code-snippet}]}
+   {:label "Last Message"
+    :key   :last-message
+    :type  :text}
+   (preview/customization-color-option)])
+
+;; Mock data
+(def sticker {:source (resources/get-mock-image :sticker)})
+(def community-avatar {:source (resources/get-mock-image :community-logo)})
+(def gif {:source (resources/get-mock-image :gif)})
+(def coinbase-community (resources/get-mock-image :coinbase))
+(def photos-list
+  [{:source (resources/get-mock-image :photo1)}
+   {:source (resources/get-mock-image :photo2)}
+   {:source (resources/get-mock-image :photo3)}
+   {:source (resources/get-mock-image :photo1)}
+   {:source (resources/get-mock-image :photo2)}
+   {:source (resources/get-mock-image :photo3)}])
+(def clojure-example
+  "(defn request->xhrio-options
```

Comment:
what is this string? is it relevant? probably we can delete it?

Reply:
The string is just a mock string for the code snippet. @J-Son89

Reply:
Oh I see 😅